libs/ledger/include/ledger/chain/address.hpp

```diff
@@ -102,6 +104,24 @@ class Address
   ConstByteArray display_;  ///< The display representation
 };
 
+template <typename T>
```

Comment:
Moved serializer into same file rather than seperate class due to header inclusion ordering annoyance